debugs.c

```diff
@@ -13,6 +13,7 @@ void enable_debugs ()
 	logstream = stderr;
 
 #define TEST(x)	access (DEBUGDIR #x, F_OK) == 0;
+//#define TEST(x)	1;
```

Comment:
excess? or really needed?

Reply:
That was needed to find the problems when compiling pyvm, probably it's better to add a command line to activate it on demand.